doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -59,13 +59,31 @@
 <a name="3.2.2"></a>
 ### 3.2.2 Container Networking Services
 
-> This chapter should discuss networking related considerations of container network services, like:
-> * Ingress and any limitations on protocol or raw interface support
-> * Dependencies on external loadbalancers
-> * Regular service mesh related considerations
-> * Network Service Mesh related considerations
-> * Number of interfaces and IP addresses to be supported by a pod
-> * IPv6 single and dual stack needs
+Because `req.inf.ntw.01` requires the architecture to support CNI and `req.inf.ntw.16` requires the capability to attach several network interfaces to the pods the architecture must support a CNI metaplugin/CNI multiplexer.
+
+A CNI metaplugin/CNI multiplexer is capable to attach several interfaces, using different other CNI plugins, to a pod. The different network charastheristics of the interfaces, which require different networking technologies require different CNI plugins.
+
+Inter node communication required by `req.inf.ntw.02` must be served by a CNI plugin with basic networking capabilities, like Calico or Flannel.
```

Comment:
I wouldn't use the word "basic". Calico does provide some extra features.
Also I think every CNI should be capable of setting-up interface through which inter-node communication is possible :)

Instead I would mention that there needs to be at least one network available in the cluster which complies with the default K8s networking assumptions, and the CaaS infra components are reachable through it.
And this network can be an overlay network too, as latency does not matter here.

Also I would not mention Flannel, as it is quickly falling out of grace due to it not supporting NetworkPolicies. Calico / Weave / Cilium are generally seem as "better" options for this network

Reply:
I think "basic" here means more about realizing network connectivity of inter node. Overlay network can be included. And also I think we have no need to mention a specific CNI plugin here just to describe its capabilities. For network policy should we make another network requirement if CNI needs to support?

Reply:
I've made an attempt to correct this.